modules/configuration/src/main/java/org/apache/ignite/configuration/ConfigurationChanger.java

```diff
@@ -44,68 +47,158 @@
     private final ForkJoinPool pool = new ForkJoinPool(2);
 
     /** Map of configurations' configurators. */
-    private Map<RootKey<?>, Configurator<?>> registry = new HashMap<>();
+    @Deprecated
```

Comment:
Can we remove this field altogether? Seems odd to have something offically deprecated in alpha

Reply:
Sure, we will delete it, it's left for tests now, because there's no substitution for configurators yet.